azure/src/services/azureFunctionCloudService.ts

```diff
@@ -43,12 +56,15 @@ export class AzureFunctionCloudService implements CloudService {
    * @param name Name of function to invoke
    * @param fireAndForget Wait for response if false (default behavior)
    * @param payload Body of HTTP request
+   * @param headers Headers of the context
+   * @param params StringParams with values for URL
    */
   public async invoke<T>(
     name: string,
     fireAndForget,
     payload: any = null,
-    headers: StringParams = new StringParams()
+    headers: StringParams = new StringParams(),
+    params?: StringParams =  new StringParams()
```

Comment:
Remove `?` its not required if you already defined a default value.

Also, remove extra space between `=` and `new`